spec/lib/shrine/plugins/validate_virus_free_spec.rb

```diff
@@ -35,7 +35,7 @@ def errors
       context 'while in development' do
         it 'logs an error message if clamd is not running' do
           expect(Rails.env).to receive(:development?).and_return(true)
-          expect(Shrine.logger).to receive(:error).with(/PLEASE START CLAMD/)
+          expect(Rails.logger).to receive(:error).with(/PLEASE START CLAMD/)
```

Comment:
Shrine.logger in a Rails application simply delegates to `Rails.logger`, so we are changing it for simplicity and homogeneity.